src/components/molecules/ButtonForm.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import Button from 'src/components/atoms/Button';
+
+type ButtonFormTypes = {
+  text: string,
+  label: string,
+  testId: string
+}
+const ButtonForm = ({ text, label, testId }: ButtonFormTypes) => {
```

Comment:
Why is this component called ButtonForm?

Reply:
I don't think this is just for forms, we could actually use this for other scenarios, Don't you think?

Reply:
I removed the component because I'm aware of its usability and it doesn´t have a good impact on the entire application's performance.